classes/class.ilTestCronConfigGUI.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require_once __DIR__ . "/../vendor/autoload.php";
+
+use iLUB\Plugins\TestCron\Helper\DIC;
+
+/**
+ * Class ilTestCronConfigGUI
```

Comment:
Rename ilCleanUpSessionsCron... for all classes...

Reply:
✔